docs/deploy_private.md

```diff
@@ -0,0 +1,45 @@
+
+# Deploying with private access
+
+If you want to disable public access when deploying the Chat App, you can do so by setting `azd` environment values.
+You should set these values before running `azd up`. Once you've set them, return to the [deployment steps](../README.md#deploying).
```

Comment:
Not sure we should have the "return" sentence here, given the difference in process?

Reply:
removed